cypress/README.md

```diff
@@ -15,6 +15,29 @@ yarn cypress-run
 
 For more usage, refer to [Cypress Commands](https://docs.cypress.io/guides/guides/command-line).
 
+### Setup your Cypress env variables
+- the Cypress suite requires an environment variable that should be stored in your `.env.development` and not committed to git.
```

Comment:
can it be .env.local instead of .env.development?

Reply:
actually was about to use .env.local. Updated.